front/css.php

```diff
@@ -59,6 +60,17 @@
 // Ensure warnings will not break CSS output.
 ErrorHandler::getInstance()->disableOutput();
 
+// If a custom theme is requested, we need to get the real path of the theme
+if (isset($_GET['file']) && isset($_GET['is_custom_theme']) && $_GET['is_custom_theme']) {
+    $theme = ThemeManager::getInstance()->getTheme(preg_replace('/\.scss$/', '', $_GET['file']));
```

Comment:
Why adding the `.scss` extention to the `file` parameter to then remove it?

Reply:
The addition of the '.scss' was required to obtain a link with the css_path filter, but on reflection, this wasn't the right solution.